orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -0,0 +1,417 @@
+import functools
```

Comment:
Should we have copyright statements in this file (and the other Python source files)?

training_agent.py seems to have a good example of one.

Reply:
some other python source files still don't have a copyright statement

---
In reply to: [593323460](https://github.com/microsoft/onnxruntime/pull/6973#discussion_r593323460) [](ancestors = 593323460)